lib/playlist.rb

```diff
@@ -0,0 +1,42 @@
+class Playlist < ActiveRecord::Base
+  validates_presence_of :created_at
+
+  has_many :playlist_songs
+  has_many :songs, through: :playlist_songs
+
+  def alphabetized_hash
+    songs.group_by{ |s| s.alpha_categorize }
+  end
+
+  def alphabet_winners_hash
+    Hash[self.alphabetized_hash.map { |k,a| [k,a.max_by {|song| song.total_votes_current_week }]}]
+  end
+
+  def Playlist.current
+    Song.all.select { |song| song.suggested_date.week_number == Time.now.week_number }
+  end
+
+  def Playlist.current_alpha_hash
+    Hash[(Playlist.current.group_by{ |s| s.alpha_categorize }).sort]
```

Comment:
Seems like there's some duplication between this and the `SongList` class. Could we reuse that here: `SongList.new(Playlist.current).get_list`?